src/autoscaler/configutil/cf.go

```diff
@@ -171,6 +180,17 @@ func (vc *VCAPConfiguration) addConnectionParams(service *cfenv.Service, dbName
 	return nil
 }
 
+func (vc *VCAPConfiguration) addConnectionParams(service *cfenv.Service, dbName string, parameters url.Values) error {
+	// if service.Tags contains "postgres" then add the connection parameters
+	if contains(service.Tags, "mysql") {
+		return nil
+		// TODO: add support for MySQL TLS enabled connections
```

Comment:
is this really a TODO ?

Reply:
I'm fine with not supporting TLS to MySQL on CF. 

I wanted to remove MySQL support, but last time I asked there were still some installations: https://github.com/cloudfoundry/app-autoscaler-release/discussions/2900